plugins/headlamp-plugin/package.json

```diff
@@ -4,10 +4,13 @@
   "description": "The needed infrastructure for building Headlamp plugins.",
   "bin": "bin/headlamp-plugin.js",
   "scripts": {
-    "prepack": "(node -e \"if (! require('fs').existsSync('./types/components')) {process.exit(1)} \" || (echo 'types dir is empty. Remember to run `npm run build` before packing' && exit 1))",
-    "build": "cd ../../frontend && tsc --build ./tsconfig.plugins.json"
+    "prepack": "(node -e \"if (! require('fs').existsSync('./lib/components')) {process.exit(1)} \" || (echo 'lib dir is empty. Remember to run `npm run build` before packing' && exit 1))",
+    "build": "cd ../../frontend && tsc --build ./tsconfig.plugins.json && cd ../plugins/headlamp-plugin && npx shx cp -r ../../frontend/src/assets lib/ && npx shx cp -r ../../frontend/src/resources lib/ && npx shx cp -r ../../frontend/src/i18n/locales lib/i18n"
   },
   "dependencies": {
+    "@apidevtools/json-schema-ref-parser": "^9.0.9",
```

Comment:
If this is mostly a copy of the dependencies we have in the frontend/ folder, should we try to have this file be generated from a template one? So we only have to care about dependencies in one place.

Reply:
There's a script now to sync dependencies. There's also a ci check to make sure they're synced.